src/Recruit.Storage.Client/Core/Entities/Vacancy.cs

```diff
@@ -1,7 +1,13 @@
+﻿using System;
+
 namespace Esfa.Recruit.Storage.Client.Core.Entities
 {
     public class Vacancy
     {
+        public Guid Id { get; set; }
         
+        public int VRN { get; set; }
```

Comment:
I am not against having a shorter name as discussed yesterday. Since we will have a collection for `Vacancy` I can understand ommiting the word Vacancy from the field but maybe we call it `Reference` instead, that way from looking at the database it is easier to interpret what that field is for without knowing about the codebase. `vrn` is a bit ambiguous but I understand your keystrokes reasoning.